taskfiles/Golang.yml

```diff
@@ -0,0 +1,51 @@
+version: '3'
+
+vars:
+  OPSLEVEL_GO_PKG: "github.com/opslevel/opslevel-go/v2024"
+
+tasks:
+  has-latest-opslevel-go:
+    desc: Check if latest release of opslevel-go in go.mod
+    internal: true
+    silent: true
```

Comment:
i'm pretty sure marking this silent means the echo statements you have in `cmds` won't ever be seen.

If you want to keep it silent then I think using `status` or `preconditions`

Reply:
`silent` here only prevents the yellow `task: <commands and such>` parts from being printed. The actual echo output still displays. That said, `silent:` is not needed and is a holdover from early days tinkering

Reply:
Using `status` is def the way to go 🙌